builder/bundles.go

```diff
@@ -230,16 +230,18 @@ func resolveFiles(numWorkers int, set bundleSet, bundleRepoPkgs map[string]map[s
 
 	fileWorker := func() {
 		for bundle := range bundleCh {
+			var e error
 			fmt.Printf("processing %s\n", bundle.Name)
 			// Resolve files for this bundle, passing it the map of repos to packages
-			err = resolveFilesForBundle(bundle, bundleRepoPkgs[bundle.Name], packagerCmd)
-			if err != nil {
+			e = resolveFilesForBundle(bundle, bundleRepoPkgs[bundle.Name], packagerCmd)
+			if e != nil {
 				// break on the first error we get
 				// causes wg.Done to be called and the worker to exit
 				// the break is important so we don't overflow the errorCh
-				errorCh <- err
+				errorCh <- e
 				break
 			}
+
```

Comment:
extra whitespace?

Reply:
yeah I formatted with go fmt just now and repushed :) thx